src/main/drivers/serial_tcp.h

```diff
@@ -41,7 +42,7 @@ typedef struct {
     uint8_t id;
 } tcpPort_t;
 
-serialPort_t *serTcpOpen(int id, serialReceiveCallbackPtr rxCallback, void *rxCallbackData, uint32_t baudRate, portMode_e mode, portOptions_e options);
+serialPort_t *serTcpOpen(serialPortIdentifier_e id, serialReceiveCallbackPtr rxCallback, void *rxCallbackData, uint32_t baudRate, portMode_e mode, portOptions_e options);
```

Comment:
Already fixed in master

Reply:
this doesn't appear to be in 31bd403446961b89d98e0e5577f2c46e9f83e0a1

Reply:
![image](https://github.com/user-attachments/assets/a0356cf9-9038-4ce9-904c-cd8ea24295e3)

Reply:
that is the .c file, my patch is for the header https://github.com/betaflight/betaflight/blob/master/src/main/drivers/serial_tcp.h#L44. The declaration and the definition are mismatched causing the compile error

Reply:
Ah see now. Function declaration has not been updated. Good catch.